### ISSUE TITLE
[Accessibility], WCAG 1.1.1. The name of the service is not clear to screen readers. The structure of the page is not logical in mobile view and with screen readers, recommended to fix

### ISSUE THREAD

**Issue by Eiskore** [NatLibFi/Skosmos#1976]
## URL address of the page where you encountered the problem
https://test.dev.finto.fi/en/

## Description of the problem
Screen reader users do not get the informationl from the beginning/top of the page that this is a Finto service. The Finto logo is centrally displayed to sighted users and indicates the name of the service, but there is no alternative text to the image. However, the hero image area has a hidden screen-reader-only H1 heading "Skosmos". The name Skosmos, on the other hand, is not conveyed to sighted users.

The Welcome to Finto.fi -section is available in the desktop view on the right side of the service. It pops up for sighted desktop users right at the beginning of the page and acts as an explainer/welcome text for the service. The section also provides instructions on how to use the site.
However, the section does not work as well with screen readers and in mobile view, because then the section is read and shown only at the end of the page, after a long column on the left.

**FIX**: The name of the service Finto should also be displayed on screen readers. 

The most **recommended** fix is a structural change to the front page. Especially considering mobile users and screen reader users, the structure of the front page would be much more functional if Welcome to Finto.fi! section would be placed as the first section in the main content, in which case "Welcome to Finto.fi!" would serve as the main title of the page and the section below would serve as a guiding section for the use of the page.

If such an extensive structural change to the page is not desired, at least screen reader users should be better communicated about the name of the service. Then change the hidden H1 title of the page to match the service name "Finto".

## Additional information (e.g. screenshots) about the problem

<img width="1004" height="375" alt="Image" src="https://github.com/user-attachments/assets/acbbd7eb-2995-4aac-83e7-578183289f94" />

<img width="1004" height="443" alt="Image" src="https://github.com/user-attachments/assets/e0d56400-1d3b-4b34-b53a-024c8931b4e2" />

<img width="250" height="732" alt="Image" src="https://github.com/user-attachments/assets/2230aa21-6ad4-4e7a-9888-3a5694d2daf2" />
Welcome to Finto! -section is at the end of the page on mobile view and to screen reader users.

<img width="1004" height="616" alt="Image" src="https://github.com/user-attachments/assets/bcb9b9c3-a7bd-467b-a6cf-9d91aeb5dbff" />
It is recommended to place "Welcome to finto! -section" as a first section in the main content.

**Comment by Vainonen** [comment]
h1 element is fixed with PR #2018 and welcome text is hidden in mobile layout, but navigating and showing/reading order of vocabulary list, welcome text and news is yet to be decided. Hahmo layout has a suggestion of mobile navigation menu showing (only) when hamburger menu is clicked open.

**Comment by tinskuu** [comment]
This issue is partly fixed.
